src/main/java/de/ait/secondlife/controllers/OfferController.java

```diff
@@ -134,9 +128,16 @@ public ResponseEntity<OfferResponseWithPaginationDto> getByUserId(
             description = "Creating a new offer and saving it in the database"
     )
     @ApiResponses(value = {
-            @ApiResponse(responseCode = "201", description = "Entity created",
-                    content = @Content(mediaType = "application/json",
-                            schema = @Schema(implementation = OfferResponseDto.class))),
+            @ApiResponse(responseCode = "201", description = "Offer created",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = OfferResponseDto.class))),
+            @ApiResponse(responseCode = "400", description = "Invalid input",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ValidationErrorsDto.class))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized",
```

Comment:
It seems to me that everywhere there will be a similar situation with 401 error. this is our internal protection, but for users (and QA) it will be unreachable, just like 500